controllers/dspipeline_params.go

```diff
@@ -646,6 +647,28 @@ func (p *DSPAParams) ExtractParams(ctx context.Context, dsp *dspa.DataSciencePip
 			}
 		}
 
+		if p.APIServer.CustomKfpLauncherConfigMap != "" {
+			cm, err := util.GetConfigMap(ctx, p.APIServer.CustomKfpLauncherConfigMap, p.Namespace, client)
+			if err != nil {
+				// If the custom kfp-launcher ConfigMap is not available, that is ok. Just log it and move along.
```

Comment:
Wouldn't it be better to leave it explicit to users (fail fast) by returning an error?

Reply:
I'm not sure. It didn't feel like a fatal error that should block the rest of the rollout of the DSPA. I could be convinced that it should be fatal.

Reply:
My concern is that users may not notice the log message and take too long to discover that they aren't using the proper config.

Reply:
fixed